x-pack/plugins/fleet/server/collectors/agent_policies.ts

```diff
@@ -25,37 +47,48 @@ export const getAgentPoliciesUsage = async (
   esClient: ElasticsearchClient,
   abortController: AbortController
 ): Promise<AgentPoliciesUsage> => {
-  try {
-    const res = await esClient.search(
-      {
-        index: AGENT_POLICY_INDEX,
-        size: ES_SEARCH_LIMIT,
-        track_total_hits: true,
-        rest_total_hits_as_int: true,
+  const res = await esClient.search<{}, AgentPoliciesUsageAggs>(
+    {
+      index: AGENT_POLICY_INDEX,
```

Comment:
If we query on the `.fleet-policies` index, won't we pick up policies that have already been deleted from the UI? I think instead we should be doing this query on the agent policy SOs instead.

Reply:
You're right, I incorrectly thought that we remove from that index on policy deletion, I'll close this PR and put up a new one using SOs